app/pages/TransactionForm.tsx

```diff
@@ -185,6 +196,9 @@ export const TransactionForm = ({
       setIsReceiveInputActive(true);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
+
+    // Setting first render to false
+    isFirsRender.current = false;
```

Comment:
move the `// eslint-disable-next-line react-hooks/exhaustive-deps` to be just before the dependency array